spotify-api.el

```diff
@@ -176,7 +176,9 @@ JSON response."
 
 (defun spotify-get-item-uri (json)
   "Returns the uri from the given track/album/artist object."
-  (gethash 'uri json))
+  (if (gethash 'linked_from json)
```

Comment:
trackid:s are different depending on what context they are in.  The linked_from has the element in the current context (I hope).